django/db/models/expressions.py

```diff
@@ -522,6 +526,24 @@ def copy(self):
         copy.extra = self.extra.copy()
         return copy
 
+    @property
+    def lhs(self):
+        """
+        This property exists for compatibility with the previously
```

Comment:
I'm on the fence with deprecation. It causes no harm being here, and is no burden to maintain. It was also public API in the transform literature, and is still used for Lookups. If we deprecate, anyone that has written a custom transform is likely to have to change their code.

So, we should either deprecate it, or remove the comment. @akaariai have you got a preference here? I'm leaning towards removing the comment and leaving the `lhs` property as it is. If we go this route, should we also document it?